app/src/main/res/values/strings.xml

```diff
@@ -85,96 +85,96 @@
     <string name="mark_as_read_one_day">Mark as Read More Than 1 Day</string>
     <string name="mark_as_read_three_days">Mark as Read More Than 3 Days</string>
     <string name="mark_as_read_seven_days">Mark as Read More Than 7 Days</string>
-    <string name="one_day">1d</string>
-    <string name="three_days">3d</string>
-    <string name="seven_days">7d</string>
+    <string name="one_day">1 d</string>
+    <string name="three_days">3 d</string>
+    <string name="seven_days">7 d</string>
     <string name="close">Close</string>
-    <string name="get_new_updates">Get new updates</string>
-    <string name="get_new_updates_desc">Version %1$s has been released</string>
+    <string name="get_new_updates">Look for new versions of the app</string>
+    <string name="get_new_updates_desc">Version %1$s is available</string>
     <string name="in_coding">In coding</string>
     <string name="coming_soon">Coming soon</string>
     <string name="accounts">Accounts</string>
     <string name="accounts_desc">Local, FreshRSS</string>
-    <string name="color_and_style">Color &amp; style</string>
+    <string name="color_and_style">Color and style</string>
     <string name="color_and_style_desc">Theme, color style, font size</string>
     <string name="interaction">Interaction</string>
     <string name="interaction_desc">On start, haptic feedback</string>
     <string name="languages">Languages</string>
     <string name="languages_desc">English, 简体中文, Deutsch</string>
-    <string name="help_translate">Help us translate</string>
-    <string name="use_device_languages">Use Device Language</string>
+    <string name="help_translate">Help translate the app</string>
+    <string name="use_device_languages">Default</string>
     <string name="english" translatable="false">English</string>
     <string name="chinese_simplified" translatable="false">简体中文</string>
     <string name="german" translatable="false">Deutsch</string>
-    <string name="tips_and_support">Tips &amp; support</string>
-    <string name="tips_and_support_desc">About, open source</string>
+    <string name="tips_and_support">Tips and support</string>
+    <string name="tips_and_support_desc">About, copylefted libre software</string>
     <string name="welcome">Welcome</string>
-    <string name="agree_terms">Before you can continue, you need to agree to Read You\'s Terms of Service and Privacy Policy.</string>
-    <string name="view_terms">View the &lt;i&gt;&lt;u&gt;Terms of Service &amp; Privacy Policy&lt;/u&gt;&lt;/i&gt;</string>
+    <string name="agree_terms">Read and agree to Read You\'s Terms of Service and Privacy Policy to continue.</string>
```

Comment:
In my view this could be the link and the other one scrapped.
The policy should follow the app IMO, but it isn't really needed.
Put it in About?